models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      recipient_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      services_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
```

Comment:
These aren't needed anymore and will be automatically generated by the associations.